astroplan/plots/plots.py

```diff
@@ -0,0 +1,294 @@
+        time = time + np.linspace(-12, 12, 100)*u.hour
+
+    # Calculate airmass
+    altitude = observer.altaz(time, target).alt
+    airmass = (1.0/np.cos(90*u.deg - altitude))
+
+    # Some checks & info for labels.
+    assert len(time) == len(airmass)
+
+    if hasattr(observer, 'name') is False:
+        observer_name = 'No Observer/Location Name'
+    else:
+        observer_name = observer.name
+
+    if hasattr(target, 'name') is False:
+        target_name = str('%.3f' % (random.random()))
+    else:
+        target_name = target.name
+
+    observe_date = time[0].datetime.strftime('%Y-%m-%d')
+    observe_timezone = 'UTC'
+
+    # Plot data.
+    ax.plot_date(
+        time.plot_date,
+        airmass,
+        linestyle=style_kwargs['linestyle'],
+        color=style_kwargs['color'],
+        label=target_name,
+        )
+    ax.figure.autofmt_xdate()
+    ax.invert_yaxis()
+
+    # Set labels, title, legend, etc.
+    ax.set_ylabel("Airmass")
+    ax.set_xlabel("Hour - "+observe_timezone)
+    ax.set_title("Airmass vs Time | "+observer_name+" | "+observe_date+" "+observe_timezone)
+    ax.legend(shadow=True)
+
+    return ax
+
+
+def plot_parallactic(target, observer, time, ax=None, style_kwargs=None, dark_plot=False):
+    """
+    TODO:
+        1) dark_plot style
+        2) Parallactic angle equation? Move this to separate function?
+        3) observe_timezone -- update with info from observer?
+        4) Dealing with NaN or invalid values.
+
+    Returns an ax object with a parallactic angle vs time plot.
+
+    If an ax object is passed in, plots an additional parallactic angle
+    plot on top.
+    Otherwise, creates a new ax object with a parallactic angle plot.
+
+    Parameters
+    ----------
+    target: `~astroplan.FixedTarget`
+        The celestial body of interest.
+
+    observer: `~astroplan.Observer`
+        The person, telescope, observatory, etc. doing the observing.
+
+    time : `~astropy.time.Time`
+        An collection of times to use in generating data to plot.
+        Can be scalar (or len(time)=1).
+
+    ax : `~matplotlib.axes.Axes` or None
+        Optional.
+
+    style_kwargs : WHAT TYPE IS DICTIONARY? or None
+        Optional.
+        A dictionary with plotting style options.
+
+    dark_plot: Boolean.
+        Optional. False (default) or True.
+    """
+
+    # Set up axes & plot styles if needed.
+    if ax is None:
+        ax = plt.gca()
+    if style_kwargs is None:
+        # Need to add more style options for ax's with existing plots?
+        style_kwargs = {'linestyle': '-', 'color': 'r'}
+    #if dark_plot is True:
+        #Add dark plot stuff here.
+
+    # If Time object is scalar, pad time window.
+    if time.isscalar:
+        time = time + np.linspace(-12, 12, 100)*u.hour
+
+    # Calculate parallactic angle.
+    declination = target.dec
+    latitude = observer.location.latitude
+    azimuth = observer.altaz(time, target).az
+    hour_angle = target.ra
```

Comment:
this isn't right: RA and HA are not the same thing.  more generally, I think the actual computation of the parallactic angle is something that is going to be on `Observer`, right @bmorris3 ?  So you could just do:
```
parallactic_angle = obs.parallactic_angle(time, target)
```
```